src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/Ncwms.java

```diff
@@ -104,7 +109,15 @@ private void proxyWmsRequest(HttpServletRequest request, HttpServletResponse res
         try {
             @SuppressWarnings("unchecked")
             Map<String, String[]> wmsParameters = new HashMap<String, String[]>(request.getParameterMap());
-            wmsParameters.remove("TIME");
+
+            // Some collections such as SRS are indexed with a timestamp which doesn't match
```

Comment:
Looking at changing the harvester - this is an interim fix as it will take quite some time to reharvest all the SRS data